lib/Viroverse/Controller/Homepage.pm

```diff
@@ -20,6 +20,11 @@ sub index : Chained('base') PathPart('') Args(0) {
     my ($self, $c) = @_;
     my $active     = $c->model("ViroDB::Project")->active;
 
+    # If we have configured a root redirection, follow it
+    if (Viroverse::Config->conf->{redirect_root_to}) {
+        return $c->response->redirect(Viroverse::Config->conf->{redirect_root_to});
```

Comment:
As implied in my other comment you can probably use `RedirectToUrl` here for slightly fewer characters. I don't think any reasonable configuration would want to redirect the root away from Viroverse, since after all if you do you aren't in the app anymore, but I don't think it's a security issue if it is technically possible.